ui/src/containers/configs/NewConfig.js

```diff
@@ -74,6 +91,12 @@ class NewConfig extends Component {
     this.removeStreamConfig = this.removeStreamConfig.bind(this);
     this.updateConfigSpec = this.updateConfigSpec.bind(this);
     this.updateTab = this.updateTab.bind(this);
+    // payloadEditor specific functions
```

Comment:
Thanks for grouping the functions logically. That helps a lot with navigating them 👍